src/lazyload-image.ts

```diff
@@ -62,9 +62,10 @@ function setLoadedStyle(element: HTMLElement) {
     return element;
 }
 
-export function lazyLoadImage(image: HTMLElement, imagePath: string, errorImgPath: string, offset: number) {
+export function lazyLoadImage(image: HTMLElement, imagePath: string, defaultImagePath: string, errorImgPath: string, offset: number) {
     return (scrollObservable: Observable<Event>) => {
         return scrollObservable
+            .do(() => setImage(image, defaultImagePath))
```

Comment:
This will trigger a rerender of the image for every scroll event. I think it is better to move `setImage` outside the Observable:
```js
setImage(image, defaultImagePath);
return scrollObservable
  .filter(() => isVisible(image, offset))
```